iina/FFmpegController.m

```diff
@@ -307,4 +307,29 @@ - (void)saveThumbnail:(AVFrame *)pFrame width:(int)width height:(int)height inde
   }
 }
 
++ (double)probeVideoDurationForFile:(nonnull NSString *)file
```

Comment:
Why is this written in C?

Reply:
I want to keep all FFmpeg related calls in `FFmpegController`, and Obj-C is a little bit easier to use when interacting with C API.